projects/packages/videopress/src/class-videopresstoken.php

```diff
@@ -0,0 +1,202 @@
+<?php
+/**
+ * VideoPress Uploader
+ *
+ * @package automattic/jetpack-videopress
+ */
+
+namespace Automattic\Jetpack\VideoPress;
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+use Jetpack_Options;
+use WP_REST_Response;
+
+/**
+ * VideoPress token utility class
+ */
+class VideoPressToken {
```

Comment:
I think the class name should follow the convention, something like `VIDEOPRESS_REST_API_V1_...`

Reply:
I agree with @retrofox.
The class is not a simple token handler (e.g. `Automattic\Jetpack\Connection\Tokens`), but actually a REST endpoint, so we should name it accordingly.

Reply:
Done [9be629b](https://github.com/Automattic/jetpack/pull/26406/commits/9be629b56940b8c46ad3e07753ecfcb10792a63c)